File: src/Employee.java
import java.util.ArrayList;

class Employee {
    private String name;
    private String lastName;
    private ArrayList<BankAccount> accounts;

    public Employee(String name, String lastName, long accountNumber, char type) {
        this.name = name;
        this.lastName = lastName;
        this.accounts = new ArrayList<>();
        accountNumber=BankAccount.CreacionaccountNumber();
        addAccount(accountNumber, type);
    }

    public void addAccount(long accountNumber, char type) {
        BankAccount.CreacionaccountNumber();
        accounts.add(new BankAccount(accountNumber, type));
    }

    public String getName() {
        return name;
    }

    public String getLastName() {
        return lastName;
    }

    public ArrayList<BankAccount> getAccounts() {
        return accounts;
    }
}

File: Employee.py
from BankAccount import BankAccount

class Employee:
    def __init__(self, name, last_name, account_number, account_type):
        self.name = name
        self.last_name = last_name
        self.accounts = []
        self.add_account(account_number, account_type)

    def add_account(self, account_number, account_type):
        self.accounts.append(BankAccount(account_number, account_type))

    def get_name(self):
        return self.name

    def get_last_name(self):
        return self.last_name

    def get_accounts(self):
        return self.accounts